### PR TITLE
Optional meta generator & comments

### PR DESCRIPTION
**Pull request by maveille**
Meta generator and comments can be optional with {% seo footprint=false %} param

### CODE REVIEW

**Comment by maveille** [comment]
Anyone can help ? i dont know why its write that dont work, i have test and it is

File: docs/advanced-usage.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ If for some reason, you don't want the plugin to output `<title>` tags on each p
 ```
 <!-- {% endraw %} -->
 
+### Disabling `generator tag and comments` output
+
+If for some reason, you don't want the plugin to output `<!-- Begin Jekyll SEO tag v2.5.0 --> <meta name="generator" content="Jekyll vx.x.x" /><!-- End Jekyll SEO tag -->` tags on each page, simply invoke the plugin within your template like so:
```
Comment (benbalter) [inline docs/advanced-usage.md]: > If for some reason

Per @ashmaroli in https://github.com/jekyll/jekyll-seo-tag/issues/307#issuecomment-428299857, I believe the maintainers' consensus was that this feature should not be implemented within the project absent a compelling use case to justify the necessary added complexity.

**Comment by DirtyF** [comment]
Closing as wontfix.

**Comment by pongstr** [comment]
@DirtyF at least state a reason why it's labeled as **wont fix**

**Comment by ashmaroli** [comment]
@pongstr @maveille Why is the presence of "the footprint" such an issue..?? Why are you so driven on having it not rendered..?

**Comment by maveille** [comment]
Like they dont want my push, i have clone for me so if you want you can see on my repo @pongstr

**Comment by pongstr** [comment]
@ashmaroli

> Why is the presence of "the footprint" such an issue..?? 
> Why are you so driven on having it not rendered..?

I'm wondering what makes you think that **I'm driven to not having it rendered**? The 👎 is for closing as `wontfix` without a reason, so please don't assume.

Personally, it doesn't bother me however when a request from the company I work for asked me to have it remove, i would like to have a valid reason to my client and also understand where the source authors are coming from.

**Comment by ashmaroli** [comment]
I understand that clients can be demanding.
But the reason this was marked as `wont-fix` because *the maintainers of this project and myself don't see how having these "footprints" in an HTML page is a source of concern.*

Adding `{% if %}` blocks (and other Liquid) is a complexity that will have an impact on the overall build times (more so in large site with numerous posts, documents and pages). So, the maintainers won't accept this change unless the community provides a reason to validate the added complexity.

That said, users are free to fork the project, customize it and use the fork in their project.

**Comment by pongstr** [comment]
> That said, users are free to fork the project, customize it and use the fork in their project.

No worries buddy, done that already. I just thought there's a profound reason behind. IMO, one more if doesn't really make significant `-/+` to build time. But that's okay, I get you.

**Pull request closed without merging**